### PR TITLE
bpfman release v0.5.2

### PR DESCRIPTION
**Pull request by anfredette**

### CODE REVIEW

File: changelogs/CHANGELOG-v0.5.2.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  https://github.com/bpfman/bpfman/pull/1231
+* bpfman/lib: add function documentation by @frobware in
+  https://github.com/bpfman/bpfman/pull/1186
+* Move Fedora back to release 40 in Containerfile.bpfman.local by @anfredette in
+  https://github.com/bpfman/bpfman/pull/1235
+* bpfman: Make whether to use cosign configurable by @anfredette in
+  https://github.com/bpfman/bpfman/pull/1240
+* Fix BUILDPLATFORM redefinition issue in bpfman Containerfile by @msherif1234
+  in https://github.com/bpfman/bpfman/pull/1244
+* add build-source to fix release violations by @msherif1234 in
```
Comment (msherif1234) [inline changelogs/CHANGELOG-v0.5.2.md:22]: lets drop all konflux related works from the changeset so this and the following one

Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.2.md:22]: Will do.  I deleted all of the ones done by the konflux bot, but not the konflux related ones by you.  But it's probably good to delete those too.

File: changelogs/CHANGELOG-v0.5.2.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+The v0.5.2 release is a patch release that updates bpfman to aya version 0.13.0
+and aya-obj version 0.2.0.  This update allows bpfman to make a proper bpfman
+release that doesn't have Aya pined to a git SHA.
+
+Other Notable Changes:
+* Add support for ppc and S390
```
Comment (Billy99) [inline changelogs/CHANGELOG-v0.5.2.md:6]:
```suggestion
* Add support for ppc64le and s390x
```

Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.2.md]: Done.

**Comment by anfredette** [comment]
I also needed to manually add release tags to the dispatcher images on quay, so I updated the release doc accordingly.

File: changelogs/CHANGELOG-v0.5.2.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+The v0.5.2 release is a patch release that updates bpfman to aya version 0.13.0
+and aya-obj version 0.2.0.  This update allows bpfman to make a proper bpfman
+release that doesn't have Aya pined to a git SHA.
```
Comment (dave-tucker) [inline changelogs/CHANGELOG-v0.5.2.md:3]: s/pined/pinned/g

**Review by dave-tucker — CHANGES_REQUESTED** [review]
small release note nit

File: changelogs/CHANGELOG-v0.5.2.md (hunk 3 above)
Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.2.md]: Done.

**Review by Billy99 — APPROVED** [review]
/lgtm

**Review by msherif1234 — APPROVED** [review]

**Pull request merged**